### DISCUSSION TITLE
Gpio`s

### DISCUSSION THREAD

**Discussion by ditsch78** [ClassicDIY/SkyeTracker#32] [Q&A]
Hallo
das Programm ist gut und befindet sich bei mir in der Testphase.
Wäre es möglich die Manuelle Steuerung über Gpio`s auszuführen.

Danke im voraus

**Comment by graham22** [comment]
You will have to modify the code in Tracker.cpp to call the following
functions when a GPIO input is triggered

[image: image.png]

On Sun, May 30, 2021 at 5:18 AM ditsch78 ***@***.***> wrote:

> Hallo
> das Programm ist gut und befindet sich bei mir in der Testphase.
> Wäre es möglich die Manuelle Steuerung über Gpio`s auszuführen.
>
> Danke im voraus
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/32>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATIU6WB2EUQNZL4H36TTQH7E3ANCNFSM45ZEYTCA>
> .
>


-- 
Graham A. Ross
(613) 769-1331
